Classes/Backend/Form/ToolBox.php

```diff
@@ -839,6 +849,24 @@ public static function showMenu($pid, $name, $modName, $entries, $script = '', $
 		return $ret;
 	}
 
+    /**
+     * Returns a selector box "function menu" for a module
+     * <select> tag replaced by bootstrap classes
+     * @param $mainParams
+     * @param $elementName
+     * @param $currentValue
+     * @param $menuItems
+     * @param string $script
+     * @param string $addParams
+     * @return mixed
+     */
+    public static function getFuncMenu($mainParams, $elementName, $currentValue, $menuItems, $script = '', $addParams = '')
```

Comment:
Warum ist diese Methode public? Sie wird offenbar in `showMenu()` verwendet. Weitere öffentliche statische Methoden finde ich in dieser Klasse nicht gut. Bei `showMenu()` ist es eine Altlast, normalerweise wird die Klasse nicht (mehr) statisch verwendet.